provisioning_templates/snippet/saltstack_setup.erb

```diff
@@ -23,10 +23,12 @@ else
 fi
 <% end -%>
 
-cat > <%= etc_path %>/minion << EOF
+cat > <%= etc_path %>/minion.d/minion.conf << EOF
 <%= snippet 'saltstack_minion' %>
 EOF
 
+echo <%= @host.name %> > <%= etc_path %>/minion_id
```

Comment:
Is this the preferred path or should it be in etc_path/minion.d/minion_id, too?

Reply:
The minion_id will always reside in /etc/salt directly